packages/html/index.ts

```diff
@@ -0,0 +1 @@
+export { Web3Modal } from './src/client'
```

Comment:
Since this is the index for the package, it'd be nice to export Web3Modal as default to have a nicer import

```suggestion
export { Web3Modal } from './src/client'
export { Web3Modal as default} from './src/client'
```

Very minor, but it's what's typically expected of npm packages